pkg/ci/main.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"net/http"
+	"sort"
+	"text/template"
+	"time"
+
+	"github.com/alexellis/arkade/pkg/get"
+)
+
+func URLExists(name, url, version string) error {
+	fmt.Println("--------------->>>>>>>>>>>>>>>>>>>>>>>>>", name)
+	timeout := time.Second * 10
+	client := get.MakeHTTPClient(&timeout, false)
+
+	req, err := http.NewRequest(http.MethodHead, url, nil)
+	if err != nil {
+		return err
+	}
+
+	res, err := client.Do(req)
+	if err != nil {
+		return err
+	}
+
+	if res.Body != nil {
+		defer res.Body.Close()
+	}
+
+	if res.StatusCode != http.StatusOK {
+		return fmt.Errorf("tool %s with version %s not found", name, version)
+	}
+	return nil
```

Comment:
I think you could also check that the content-length returned is not 0?

Reply:
You mean content length == 0? As this is to check if no resource is returned from the request.